client/state/jetpack-connection-health/selectors/is-requesting-jetpack-connection-health-status.js

```diff
@@ -0,0 +1,16 @@
+import 'calypso/state/jetpack-connection-health/init';
+
+/**
+ * Returns true if we are currently making a request to retrieve the connection status. False otherwise.
+ * Returns null if the site is unknown, or there is no information yet.
+ * @param  {Object}   state    Global state tree
+ * @param  {number}   siteId   The ID of the site we're querying
+ * @returns {?boolean}          Whether the connection status is being requested
```

Comment:
Let's make sure this param reflects the truth and includes any additional field types.